documentation/modules/security/con-certificate-renewal.adoc

```diff
@@ -6,30 +6,28 @@
 = Certificate renewal and validity periods
 
 [role="_abstract"]
-Cluster CA and clients CA certificates are only valid for a limited time period, known as the validity period.
-This is usually defined as a number of days since the certificate was generated.
+Cluster CA and clients CA certificates are valid for a limited time, known as the validity period. 
+This is defined as the number of days from the date the certificate was generated.
 
-For CA certificates automatically created by the Cluster Operator, configure the validity period for certificates in the `kafka` resource using the following properties:
+For CA certificates automatically created by the Cluster Operator, configure the validity period for certificates in the `kafka` resource:
 
 * `Kafka.spec.clusterCa.validityDays` for Cluster CA certificates
 * `Kafka.spec.clientsCa.validityDays` for Clients CA certificates
 
 The default validity period for both certificates is 365 days.
-Manually-installed CA certificates should have their own validity periods defined.
+For manually-installed CA certificates, set validity through your certificate management system.
 
-When a CA certificate expires, components and clients that still trust that certificate do not accept connections from peers whose certificates were signed by the CA private key.
-The components and clients need to trust the _new_ CA certificate instead.
+When a CA certificate expires, components and clients that still trust the old certificate do not accept connections from peers whose certificates were signed by the CA private key.
+The components and clients must trust the _new_ CA certificate instead.
 
-To allow the renewal of CA certificates without a loss of service, the Cluster Operator initiates certificate renewal before the old CA certificates expire.
+To prevent loss of service, the Cluster Operator initiates certificate renewal before the old CA certificates expire.
 
 Configure the renewal period of the certificates created by the Cluster Operator in the `kafka` resource using the following properties:
 
 * `Kafka.spec.clusterCa.renewalDays` for Cluster CA certificates
 * `Kafka.spec.clientsCa.renewalDays` for Clients CA certificates
 
-The default renewal period for both certificates is 30 days.
-
-The renewal period is measured backwards, from the expiry date of the current certificate.
+The default renewal period for both certificates is 30 days from the expiry date of the current certificate.
```

Comment:
Even with Custom CA, the validity / renewal options also drive the validity and renewal of the server certificates. This might be worth mentioning here.

Especially with respect to the line 18, where users can set the CA validity in their own system. But Strimzi still drives the validity of the server certificates through these options.

Another thing worth mentioning - as it often comes up - is that when the user changes the `validityDays`, it does not immediately trigger renewal but will be only used in the next renewal.